ast/ast.go

```diff
@@ -203,9 +203,9 @@ type (
 
 	// A TextAccent node is a string of text with an accent character.
 	TextAccent struct {
-		ValuePos gotok.Pos // literal position
-		Accent   string    // the accent character, like '"'
-		Value    Expr      // the text to accent
+		ValuePos gotok.Pos    // literal position
+		Accent   token.Accent // the accent character, like "^" for circumflex
+		Text     *Text        // the text to accent; must be a single rune
```

Comment:
The parser requires this to be ast.Text, so change the type to simplify life when traversing the AST to render it.